angular.json

```diff
@@ -2274,6 +2274,7 @@
             ],
             "styles": [
               "libs/components/theme/src/lib/styles/sky.scss",
+              "libs/components/theme/src/lib/styles/_mixins.scss",
```

Comment:
Is this file actually needed? This is usually imported by a component's SCSS file.

Reply:
Removed